pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -201,32 +206,42 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 
 	// Build the NamespacedName to fetch the Filesystem and make sure it exists, if not we cannot
 	// create the subvolumegroup
-	cephFilesystem := &cephv1.CephFilesystem{}
-	cephFilesystemNamespacedName := types.NamespacedName{Name: cephFilesystemSubVolumeGroup.Spec.FilesystemName, Namespace: request.Namespace}
-	err = r.client.Get(r.opManagerContext, cephFilesystemNamespacedName, cephFilesystem)
-	if err != nil {
-		if kerrors.IsNotFound(err) {
-			return reconcile.Result{}, errors.Wrapf(err, "failed to fetch ceph filesystem %q, cannot create subvolumegroup %q", cephFilesystemSubVolumeGroup.Spec.FilesystemName, cephFilesystemSubVolumeGroup.Name)
+	// On external mode the filesystem is created externally, so we don't need to check for CRD and
+	// assume it's there
+	if !cephCluster.Spec.External.Enable {
+		cephFilesystem := &cephv1.CephFilesystem{}
+		cephFilesystemNamespacedName := types.NamespacedName{Name: cephFilesystemSubVolumeGroup.Spec.FilesystemName, Namespace: request.Namespace}
+		err = r.client.Get(r.opManagerContext, cephFilesystemNamespacedName, cephFilesystem)
+		if err != nil {
+			if kerrors.IsNotFound(err) {
+				return reconcile.Result{}, errors.Wrapf(err, "failed to fetch ceph filesystem %q, cannot create subvolumegroup %q", cephFilesystemSubVolumeGroup.Spec.FilesystemName, cephFilesystemSubVolumeGroup.Name)
```

Comment:
`cannot create subvolumegroup` to `cannot create subvolume group` to keep it consistent with other places?